interfaces/builtin/login_control.go

```diff
@@ -0,0 +1,67 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const loginControlConnectedPlugAppArmor = `
+# Description: Can control the login system service
+
+#include <abstractions/dbus-strict>
+
+# Give full access to the login service. This allows a client
+# to control everything (system reboot, shutdown, suspend, ...)
```

Comment:
As already discussed over in the upower-observe interface PR we will merge certain things into the shutdown interface for logind access. That should be enough for our use case where device management snaps need to be able to suspend a device etc.